test/integration/src/client-util.js

```diff
@@ -0,0 +1,10 @@
+const configUtil = require('./config-util');
+const revai = require('revai-node-sdk');
+
+module.exports = {
+    getClient: function (apiKey) {
```

Comment:
You can also get the api key directly. I don't see a need to parameterize with the api key.
```suggestion
    getClient: () =>  {
        const client = new revai.RevAiApiClient(configUtil.getApiKey());
    }
```

Reply:
Do we want any future test to check that the correct error is returned for an invalid api key? If not then I can make the change you mentioned.